.expeditor/verify.pipeline.yml

```diff
@@ -1,31 +1,41 @@
 steps:
 
-- label: run-lint-and-specs-ruby-2.4
+- label: run-lint-and-specs-ruby-2.6
   command:
     - bundle config set --local without docs debug
     - bundle install --jobs=7 --retry=3
     - bundle exec rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.4-stretch
+        image: ruby:2.6
 
-- label: run-lint-and-specs-ruby-2.5
+- label: run-lint-and-specs-ruby-2.7
   command:
     - bundle config set --local without docs debug
     - bundle install --jobs=7 --retry=3
     - bundle exec rake
   expeditor:
     executor:
       docker:
-        image: ruby:2.5-stretch
+        image: ruby:2.7
```

Comment:
TIL they're doing the right thing with their tags now and we can stop specifying the debian release